CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Next Release
 
+#### Enso Standard Library
+
+- [Added Statistic.Product][10122]
+
+[debug-shortcuts]:
+  https://github.com/enso-org/enso/blob/develop/app/gui/docs/product/shortcuts.md#debug
```

Comment:
btw. this link seems to be broken

Reply:
Any idea what it actually does? I just copied it from the old changelog... (I think)

Reply:
Removed